src/components/bc-modal.ts

```diff
@@ -1,50 +1,27 @@
 import {html} from 'lit';
-import {customElement, state} from 'lit/decorators.js';
+import {customElement} from 'lit/decorators.js';
 import {BitcoinConnectElement} from './BitcoinConnectElement';
 import './bc-router-outlet.js';
 import './internal/bci-connecting';
-import store from '../state/store';
 import {withTwind} from './twind/withTwind';
 import './bc-modal-header';
 import {classes} from './css/classes';
 import {closeModal} from '../api';
 
 @customElement('bc-modal')
 export class Modal extends withTwind()(BitcoinConnectElement) {
-  @state()
-  protected _closing = false;
-
-  constructor() {
-    super();
-
-    const unsub = store.subscribe((currentStore, prevStore) => {
-      if (!currentStore.modalOpen && prevStore.modalOpen) {
-        this._closing = true;
-        setTimeout(() => {
```

Comment:
setTimeout also looks dangerous here. 

I think it's good to remove this complexity.